trappy/ftrace.py

```diff
@@ -88,6 +145,13 @@ def __init__(self, name="", normalize_time=True, scope="all",
         if normalize_time:
             self.normalize_time()
 
+        if self.__class__.disable_cache: return
+
+        self._create_trace_cache()
+        for trace_class in self.trace_classes:
+            csv_file = self._get_csv_path(trace_class)
+            trace_class.write_csv(csv_file)
+
```

Comment:
Yes, I think management of that cache directory (configuring its path etc) would be a pain, I'd much rather just drop them into the current directory. There csvs are also much smaller than the actual trace so its not a big space consumer.